src/types/utils.cpp

```diff
@@ -284,6 +285,791 @@ static constexpr std::array<til::color, 256> standardXterm256ColorTable{
     til::color{ 0xEE, 0xEE, 0xEE },
 };
 
+static til::static_map<std::wstring, til::color, std::less<std::wstring>, 782> xorgAppColorTable{

```

Comment:
Is this the first time `static_map` is actully used? Glad to give it a test drive.

Reply:
And I said I was goint to use a script, turned out a simple regex is good enough. So this is the result a regex + VSCode. Enjoy it.

BTW I don't think this will need to be updated, like ever. Unlike the Unicode standard which is keep evolving. The definitaion of color would have to wait for human to evolve in the level of species. Imagine how long that would take...

Reply:
So with https://github.com/microsoft/terminal/pull/7600, are you able to simply read in the rgb.txt file instead to simplify this?

Reply:
@wsluser that would require us to ship an `rgb.txt` every place conhost exists. That's going to be a big _no_.

This **is** the use of rgb.txt (or data derived therefrom) that was called out in 7600.